Document new Content-Type header for Azure (#190)
CloudCannon/platform-documentation: articles/creating-an-azure-dam.mdx

### Attach metadata to uploaded files

> Under **Advanced Options**, there are options to attach metadata to files when they are uploaded. These options only affect files uploaded after the setting is applied. To enable these behaviour, you may need to update the "Allowed Headers" in your "Resource sharing (CORS)" settings in Azure.

Reviewer: ```suggestion
You can toggle the option to attach metadata to files when they are uploaded. Navigate to the _Advanced Options_ section under _Org Settings_ > _Assets_ > _Azure DAM_.
This option is enabled by default for new DAMs. If you have an older DAM, you may need to update the "Allowed Headers" in your "Resource sharing (CORS)" settings in Azure to enable this setting.
Enabling the metadata option will only affect files uploaded after the setting is applied. 
```

> 
> You can have a file tagged with the email address of the user who uploaded it. This will appear under an `uploadedBy` key in the Azure metadata. Your Azure CORS "Allowed Headers" needs to include `x-ms-meta-uploadedBy` for this to succeed.

Reviewer: ```suggestion
```

> You can optionally attach an extra header X-Ms-Blob-Content-Type to indicate the mime-type of uploaded files to Azure. Check the `X-Ms-Blob-Content-Type` checkbox to enable this. Your Azure CORS "Allowed Headers" needs to include `x-ms-blob-content-type` for this to succeed.

Reviewer: ```suggestion
You can optionally attach an extra header X-Ms-Blob-Content-Type to indicate the MIME type of uploaded files to Azure. Check the `X-Ms-Blob-Content-Type` checkbox to enable this. Your Azure CORS "Allowed Headers" needs to include `x-ms-blob-content-type` for this to succeed.
```

> You can optionally attach an extra header X-Ms-Blob-Content-Type to indicate the mime-type of uploaded files to Azure. Check the `X-Ms-Blob-Content-Type` checkbox to enable this. Your Azure CORS "Allowed Headers" needs to include `x-ms-blob-content-type` for this to succeed.

Reviewer: ```suggestion
```
